Content.Server/_CD/Records/CharacterRecordsSystem.cs

```diff
@@ -184,10 +183,7 @@ public void DeleteAllRecords(EntityUid player, CharacterRecordKeyStorageComponen
 
     public IDictionary<uint, FullCharacterRecords> QueryRecords(EntityUid station, CharacterRecordsComponent? recordsDb = null)
     {
-        if (!Resolve(station, ref recordsDb))
-            return new Dictionary<uint, FullCharacterRecords>();
-
-        return recordsDb.Records;
+        return !Resolve(station, ref recordsDb) ? new Dictionary<uint, FullCharacterRecords>() : recordsDb.Records;
```

Comment:
This is a very long line, I think the old if condition was clearer.

Reply:
or just simply chop it up

Reply:
that also works